packages/js/src/dashboard/components/widget-table.js

```diff
@@ -11,18 +11,18 @@ import { __ } from "@wordpress/i18n";
 /**
  * The disabled score component.
  *
- * @param {string} screenReaderLabel The screen reader label.
  * @param {string} tooltip The tooltip.
  *
  * @returns {JSX.Element} The element.
  */
-const DisabledScore = ( { screenReaderLabel, tooltip } ) => {
+const DisabledScore = ( { tooltip } ) => {
+	const uniqueId = `yst-disabled-score-tooltip-${Math.random().toString( 36 ).substring( 7 )}`;
```

Comment:
I think we shouldn't randomly create an ID that way, since it's possible we get the same ID for different scores.

It would be better to pass a `key` prop from the `Score` component (I see that the `TopPagesTable` that calls the Score component already has an index that can be used) and create the ID from that